src/pages/home/Home.jsx

```diff
@@ -0,0 +1,28 @@
+import "../../App.css";
+import Header from "../../components/header/Header";
+import Categories from "../../components/categories/Categories";
```

Comment:
suggestion: maybe you can put all the components in one index file and import

Reply:
made changes!